src/components/QuizSolve/QuizCarouselItem.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+
+import styled from '@emotion/styled';
+
+import maple from '@/assets/maple.png';
+
+import type { Quiz as QuizInterface } from '@/interfaces/Quiz';
+
+interface Props {
+  quiz: QuizInterface;
+  index: number;
+  onChangeUserAnswer: (index: number, value: string) => void;
+}
+
+/**
+ * @description
+ * QuizCarousel 내부에서 사용되는 Item입니다.
+ * ANCHOR - 재사용성이 낮으므로, 다른 유형 퀴즈를 받기 위해 변경 필요
+ * 이 컴포넌트 자체가 재사용성이 낮기 때문에 더 이상 나누는 것은 불필요
+ */
+const QuizCarouselItem = ({ quiz, index, onChangeUserAnswer }: Props) => {
+  const [clickedIndex, setClickedIndex] = useState(-1);
+  const handleClickIndex = (idx: number) => setClickedIndex(idx);
+  return (
+    <Container>
+      <Question>
+        <QuestionSign>Q.</QuestionSign>
+        <QuestionTitle>{quiz.question}</QuestionTitle>
+      </Question>
+      <FlexContainer>
+        {[
+          ['O', 'true'],
+          ['X', 'false'],
+        ].map(([key, value], idx) => (
```

Comment:
`P3`변수를 통해 가독성을 조금 더 개선할 수 있을 것 같아요.
```suggestion
  const ANSWER_CHOICES = [
    ['O', 'true'],
    ['X', 'false'],
  ];
  
   ...
   {ANSWER_CHOICES.map(([key, value], idx) => ()
```

Reply:
이것도 상수화를 통해 컴포넌트 바깥으로 뺄 수 있겠네요. 굿굿~